src/bootstrap/dist.rs

```diff
@@ -518,8 +518,7 @@ impl Step for Rustc {
                     .join("bin")
                     .join(&exe);
                 // for the rationale about this rename check `compile::copy_lld_to_sysroot`
-                let dst = image.join(libdir_relative)
-                    .join("rustlib")
+                let dst = image.join("lib/rustlib")
```

Comment:
Joining `/` on Windows is a bad idea, in case the path ever turns out to be a `//?/` path which only understands `\`.

Reply:
"lib/rustlib" used here before also it used on other places as well.